scripts/changelog.js

```diff
@@ -0,0 +1,13 @@
+const {
+  default: changelogFunctions,
+} = require("@svitejs/changesets-changelog-github-compact");
+
+const customFunctions = {
+  ...changelogFunctions,
+  getDependencyReleaseLine: async () => "",
```

Comment:
Nice work!  This is badly needed!

Reply:
OMG and I just clicked it, I had NO IDEA Github could do this!  I'm putting this in my pocket for later.